drivers: can: can_stm32_bxcan.c: Failed to exit sleep mode

adityaganesh opened the pull request:
Fixes the issue of CAN failing to exit sleep mode inside can_stm32_init() when a software reset is done

Fixes: #71142

adityaganesh commented:
Closes #71164

adityaganesh commented:
Looks like the commit message size is not as per the compliance checks . So i will create another pr with the same changes but smaller commit message

adityaganesh commented:
Looks like the commit message size is not as per the compliance checks . So i will create another pr with the same changes but smaller commit message

henrikbrixandersen commented:
> Looks like the commit message size is not as per the compliance checks . So i will create another pr with the same changes but smaller commit message

Please just fix the commit message locally and force push to the branch.

adityaganesh commented:
@henrikbrixandersen I have forced push the update with new commit message to the branch

adityaganesh committed:
drivers: can: can_stm32_bxcan.c: Failed to exit sleep mode

Fixes the issue of CAN failing to exit sleep mode
inside can_stm32_init() when a software reset is done.
For stm32 the CAN is first made to enter initialization
mode first and then made to exit the sleep mode.

Signed-off-by: Aditya Ganesh <adityaganesh2k@gmail.com>
